samples/TealiumHelper.swift

```diff
@@ -29,12 +29,11 @@ class TealiumHelper : NSObject {
         
             tealium = Tealium(config: config,
                               completion:{ () in
-                #if AUTOTRACKING
-                    self.tealium?.autotracking()?.delegate = self
-                #endif
+                                
```

Comment:
Seems clear to me, especially if you jump into the add functions code comments.  Added a one liner and more code comments to the delegate module.